src/ai/backend/common/dto/storage/field.py

```diff
@@ -1,21 +1,21 @@
 from typing import Optional
 
-from pydantic import BaseModel
+from pydantic import BaseModel, Field
 
 from ...types import VolumeID
 
 
 class VolumeMetaField(BaseModel):
-    volume_id: VolumeID
-    backend: str
-    path: str
-    fsprefix: Optional[str]
-    capabilities: list[str]
+    volume_id: VolumeID = Field(description="The unique identifier of the volume.")
+    backend: str = Field(description="The backend type of the volume.")
+    path: str = Field(description="The path of the volume.")
+    fsprefix: Optional[str] = Field(description="The filesystem prefix of the volume.")
+    capabilities: list[str] = Field(description="The capabilities of the volume.")
 
```

Comment:
The description is too general and seems meaningless, how about checking how this fields will be used in the internal operation and reorganizing the description?